File: Python/Introduccion/Calculadora.py
def suma(a, b):
    return a + b


def resta(a, b):
    return a - b


def multiplicacion(a, b):
    return a * b


def division(a, b):
    if b == 0:
        return "Error: No se puede dividir por cero."
    else:
        return a / b


print("Selecciona una operación:")
print("1. Suma")
print("2. Resta")
print("3. Multiplicación")
print("4. División")


opcion = input("Ingresa el número de la operación que deseas realizar: ")


num1 = float(input("Ingresa el primer número: "))
num2 = float(input("Ingresa el segundo número: "))


if opcion == '1':
    print("Resultado:", suma(num1, num2))
elif opcion == '2':
    print("Resultado:", resta(num1, num2))
elif opcion == '3':
    print("Resultado:", multiplicacion(num1, num2))
elif opcion == '4':
    print("Resultado:", division(num1, num2))
else:
    print("Opción no válida")

File: R/Introduccion/Calculadora.R
suma<-function(x,y) {
  return(x+y)
}

resta<-function(x,y){
  return(x-y)
}

multiplicacion<-function(x,y){
  return(x*y)
}

division<-function(x,y){
  return(x/y)
}

print("selecciona una opcion: ")
print("1.- Suma")
print("2.- Resta")
print("3.- Multiplicacion")
print("4.- Division")

opcion=as.integer(readline(prompt = "Ingresa una opcion: "))
num1=as.integer(readline(prompt = "Ingresa el primero numero: "))
num2=as.integer(readline(prompt = "Ingresa el segundo numero: "))

resultado=switch(opcion,suma(num1,num2),resta(num1,num2),multiplicacion(num1,num2),division(num1,num2))

print(paste("El resultado es: ",resultado))